src/air-development/concepts.md

```diff
@@ -0,0 +1,83 @@
+# Concepts
+
+Stwo is a universal framework for proving and verifying Algebraic Intermediate Representations (AIRs). It provides a flexible framework that allows developers to:
+
+- Create a table and define a set of constraints over it
+- Efficiently connect different parts of the table using lookups
+- Create chips (i.e. custom AIRs) for specific computational tasks and compose them together using lookups
+
+In this section, we'll introduce the concepts of AIRs, lookups, and chips, and in the next section, we'll show how to use these concepts to create a custom AIR.
+
+## AIRs
+
+An AIR is simply a table where each cell is populated by an element from a finite field and where a set of predefined constraints (i.e. equations defined over multiple cells that equal to 0) are satisfied for every row.
```

Comment:
nitpick on terminology: I think the "AIR" itself is just the set of constraints, it's applied over a set of polynomials on a domain : the table.